### DISCUSSION TITLE
Video recording on all tests

### DISCUSSION THREAD

**Discussion by cungminh2710** [codeceptjs/CodeceptJS#3336] [Q&A] [answered]
In the [docs](https://codecept.io/playwright/#video), Playwright may record videos for failed tests. This can be enabled in a config with `video: true` option. 

Is it possible to tell Playwright to record videos for all tests regardless of completeness? If not, we can discuss a solution, and I can contribute a PR

**Comment by kobenguyent** [comment, marked as answer]
keepVideoForPassedTests boolean (opens new window)? save videos for passed tests; videos are saved into output/videos folder

Have you tried this?
